core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -452,8 +473,10 @@ public void init(Properties p) throws WorkloadException {
 
     transactioninsertkeysequence = new AcknowledgedCounterGenerator(recordcount);
     if (requestdistrib.compareTo("uniform") == 0) {
-      keychooser = new UniformIntegerGenerator(0, recordcount - 1);
-    } else if (requestdistrib.compareTo("zipfian") == 0) {
+      keychooser=new UniformIntegerGenerator(insertstart,insertstart+insertcount-1);
+    } else if (requestdistrib.compareTo("ordered")==0)	{
```

Comment:
I tried to reuse the "ordered" keyword from insertorder. I can either change the sequential generator to "OrderedGenerator" or change "ordered" keyword to "sequential" keyword. Do you prefer one vs the other ?